hyper/http11/parser.py

```diff
@@ -49,7 +49,8 @@ def parse_response(self, buffer):
         if index == -1:
             return None
 
-        version, status, reason = temp_buffer[0:index].split(None, 2)
+        version, status, reason = \
+            (temp_buffer[0:index].split(None, 2) + [b''])[:3]
```

Comment:
Can we avoid the explicit line continuation here? Let's use Python's implicit line continuation within parentheses and brackets instead.